cassandra/metadata.py

```diff
@@ -2054,8 +2056,8 @@ class SchemaParserV22(_SchemaParser):
         "compression",
         "default_time_to_live")
 
-    def __init__(self, connection, timeout, fetch_size):
-        super(SchemaParserV22, self).__init__(connection, timeout, fetch_size)
+    def __init__(self, connection, timeout, fetch_size, metadata_request_timeout=None):
+        super(SchemaParserV22, self).__init__(connection, timeout, fetch_size, metadata_request_timeout)
         self.keyspaces_result = []
         self.tables_result = []
```

Comment:
`SchemaParser*` structs are not part of public interface I think, se we should be able to add new mandatory argument to `__init__`. Wouldn't it be better for `metadata_request_timeout` to be such an argument, like `timeout` is? There is less room for mistakes on the caller side with mandatory arguments, and you are always passing the argument anyway.

Reply:
done